kf_lib_data_ingest/app/cli.py

```diff
@@ -30,44 +30,32 @@ def common_args_options(func):
         type=click.Path(exists=True, file_okay=True, dir_okay=True),
     )(func)
 
-    # App settings
+    # Disable warehousing
```

Comment:
I thought order of these mattered...I think this option has to be last? Since you have it as the last parameter in the method signature

Reply:
The order just affects how they show in the help. Parameters are passed to the command callbacks as keyword arguments. That's why if you remove any of them from the function signature you'll get e.g. `TypeError: test() got an unexpected keyword argument 'resume_from'`